### PR TITLE
Allow creating new properties

### PR DESCRIPTION
**Pull request by konstin**
Properties can be created nearly the same way items are created. These are the few changes required.

### CODE REVIEW

File: wikidataintegrator/wdi_core.py [hunk 1]
```diff
@@ -834,7 +834,7 @@ def get_sitelink(self, site):
         else:
             return None
 
-    def write(self, login, bot_account=True, edit_summary=''):
+    def write(self, login, bot_account=True, edit_summary='', payload_type=u'item'):
```
Comment (sebotic) [inline wikidataintegrator/wdi_core.py]: we try to have pure Py3 code, could you adjust this accordingly?

File: wikidataintegrator/wdi_core.py [hunk 2]
```diff
@@ -854,19 +855,21 @@ def write(self, login, bot_account=True, edit_summary=''):
...
         }
+        self.wd_json_representation['datatype'] = 'string'
+
         payload = {
             'action': 'wbeditentity',
             'data': json.JSONEncoder().encode(self.wd_json_representation),
             'format': 'json',
             'token': login.get_edit_token(),
-            'summary': edit_summary
+            'summary': edit_summary,
```
Comment (sebotic) [inline wikidataintegrator/wdi_core.py]: why would this change be required?

File: wikidataintegrator/wdi_core.py (hunk 1 above)
Comment (sebotic) [inline wikidataintegrator/wdi_core.py]: furthermore, payload_type is missleading, as this is a term usually used for http post requests. In the Wikidata context, entity_type would be the right term. Could you adjust that? thx!

File: wikidataintegrator/wdi_core.py [hunk 3]
```diff
@@ -855,7 +856,8 @@ def write(self, login, bot_account=True, edit_summary='', payload_type=u'item'):
             'content-type': 'application/x-www-form-urlencoded',
             'charset': 'utf-8'
         }
-        self.wd_json_representation['datatype'] = 'string'
+        if payload_type == u'property':
```
Comment (sebotic) [inline wikidataintegrator/wdi_core.py]: same here, please remove Py2 unicode data type

**Comment by konstin** [comment]
I've rename `payload_type` and removed the `u'value'` from the parts of code I've touched, though that syntax is still remaining in many other parts of the code which are outside the scope of this pull request.

**Pull request merged**